### DISCUSSION TITLE
Bluenesia font!

### DISCUSSION THREAD

**Discussion by tyrosinase** [inkstitch/inkstitch#1799] [Show and tell]
Welp, I made another Ink/Stitch alphabet: https://silverseams.com/bluenesia-alphabet/
![Bluenesia36Demo](https://user-images.githubusercontent.com/9473609/186064062-ce5a1ebd-b683-4eb6-ba76-7109b9c88806.png)

This one's got a proper license attached, but the original font didn't have all the accented letters so I am going to have to make a shorter version of the A et al. so I can fit the accents in below the cap/x-height line. (If you don't mind them popping up above it, all the pieces are there to kitbash them.) Hopefully I'll get that done before the next Ink/Stitch release, but if I don't (or if someone else wants to tackle that), feel free to merge this in.

**Comment by kaalleen** [comment]
Oh nice, I almost missed this. Maybe we can take it in as is and update later?

**Comment by tyrosinase** [comment]
Yup, let me at least zip up  the current version (which has some more but still not all of the accented letters - again, if someone with a faster computer wants to do the assembly work, all the pieces are there but at this point my computer... thinks... about... all... the...

trims...

every...

time...

I...

move...

something...)

**Comment by tyrosinase** [comment]
Just updated the download to the most current version (still haven't combined all the accented letters but all the pieces should be there): https://ko-fi.com/s/d7ccbf9bf2

(You know, I bet I could save *just* the accents/reduced letters, build all the combinations in a file that isn't so lag-inducing, and then Frankenstein-combine the two files. But if I don't get to it, this version is complete enough for everybody who isn't French. Luckily I'm sure none of them use Ink/Stitch. 😜)

**Reply by kaalleen** [reply]
With the upcoming release fonts can be made out multiple files. So if you have speed issues, you can actually split them up. In case you are doing this, name your folder with an arrow.

**Comment by tyrosinase** [comment]
Ooh! I will do that. I have not been keeping up on the discussions here lately: are there requirements (or conventions) for the filenames within the directory?

Do you have a planned release date? We closed the house deal today so hopefully I'll have some more free time (I mean, got a ton of packing left to do but now I'm not spending all my keyboard time doing all the tasks the bank and title office kept sending me).

**Reply by lexelby** [reply]
Congratulations!! No planned date, just @kaalleen and I chatting behind the scenes along the lines of "hey, maybe we should do a release soon?" :)

**Reply by tyrosinase** [reply]
Okay, I have a multi-file version with a pretty complete (well, *much more* complete, actually-complete would kill me) set of glyphs. Not gonna put them in the Ko-Fi download because that would be confusing, but holler when you need it.

I also need to re-do NickAinley. That one is a legacy font, where originally I digitized it and just copy-pasted to build words, and then later just stuck them all in a right-arrow file without doing the proper conversion-from-TTF process, so the spacing is... creative. Not sure if I'll get to that before the release.

**Reply by kaalleen** [reply]
> Not gonna put them in the Ko-Fi download because that would be confusing, but holler when you need it.

If you think it is complete enough we can add it as soon as possible. This will have the advantage that it will have more testruns before the actual release :)

**Reply by tyrosinase** [reply]
I seem to have uploaded something to the Owncloud. It's not big on confirmations though.

It's got both the old right-arrow file (incomplete) and some complete untested files in the right-arrow directory (because I have not gotten around to trying out the new versions at all).

**Reply by kaalleen** [reply]
Yes it there :)

I tested only the files in the right-arrow folder and it looks really very nice. I spotted an extra black zigzag line in é and missed the ß, but were so happy that äüö are working (I know, all these needs for other languages :O ). One rail of the capital Ö has a wrong direction.

Nice job on this font. I really do like it!!!

**Reply by tyrosinase** [reply]
Yup, ß is in with various currency symbols and other things that I haven't done yet because they weren't in the original font at all. (Neither were all the accented characters but that was just a matter of adjusting the height of the vowels and whatnot and applying the existing symbols.) I may yet get to the from-scratch group before the release, it's not that big.

**Reply by kaalleen** [reply]
That would be nice!
I uploaded the font as is here: #2127
